src/test/java/ch/bfh/bti7081/s2018/green/models/entities/EventTest.java

```diff
@@ -0,0 +1,55 @@
+package ch.bfh.bti7081.s2018.green.models.entities;
+
+import ch.bfh.bti7081.s2018.green.models.enumerations.StaffType;
+import org.junit.Assert;
+import org.junit.Test;
+
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.EntityTransaction;
+import javax.persistence.Persistence;
+import java.sql.Timestamp;
+import java.text.ParseException;
+import java.time.LocalDateTime;
+
+public class EventTest {
+    @Test
+    public void testEvent() {
+        LocalDateTime start = LocalDateTime.of(2018, 5, 21, 13, 45);
+        LocalDateTime stop = LocalDateTime.of(2018, 5, 21, 14, 30);
+
+        Person person = new Person("Emergency", "Contact", null, null, null, null, null, null);
+        Patient patient = new Patient("Patrice", "lastname", null, null, null, null, null, null, person);
+        Staff staff = new Staff("doctor", "staff", null, null, null, null, null, null, StaffType.PSYCHIATRIST);
+
+        Event event = new Event(start, stop, "Sprechstunde weil nicht gut", "Sprechstunde", patient, staff);
+
+        EntityManagerFactory emf = Persistence.createEntityManagerFactory("pmsDB");
+        EntityManager em = emf.createEntityManager();
+        EntityTransaction tx = em.getTransaction();
+
+        // insert test record
+        tx.begin();
```

Comment:
Why don't you use the newly implemented managers? They are tested and fully functional. Is it because of achieving an independent test that does not fail if the managers are changed?

Reply:
ah, i see. :)